diagnostic.md

```diff
@@ -5,74 +5,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+A backend processes and stores data - allows for persistance and shared workload
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Controller layer
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Model layer
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Good question, not sure I know why
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+CREATE
+READ
+UPDATE
+DESTROY
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+Index
+Show
+Create
+Update
+Patch
```

Comment:
Whoops, Update is similar to Patch, while the last one should have been Destroy.